server/controllers/items/by_entities.js

```diff
@@ -10,50 +10,12 @@ const sanitization = {
 }
 
 const controller = async params => {
-  return getEntitiesItems(params)
-  .then(addAssociatedData)
+  const { uris, reqUserId } = params
+  const foundItems = await items_.byEntities(uris)
+  const authorizedItems = await filterVisibleDocs(foundItems, reqUserId)
+  const page = paginate(authorizedItems, params)
```

Comment:
are we sure we can not do a more efficient request to the db by including a pagination within the view ? This looks costly, makes me sad a bit.